python_modules/dagster-webserver/dagster_webserver_tests/test_app.py

```diff
@@ -152,7 +152,7 @@ def test_successful_host_dagster_ui_from_workspace():
                 log_level="warning",
             )
 
-        assert server_call.called_with(mock.ANY, host="127.0.0.1", port=2343, log_level="warning")
+        server_call.assert_called_with(mock.ANY, host="127.0.0.1", port=2343, log_level="warning")
```

Comment:
this was fun, CPython in 3.12 got smart enough to realize that this was just asserting that a Mock object was not None (always true), instead of the intended behavior